From caa3ae31681269a4e7826502019f1831050953a6 Mon Sep 17 00:00:00 2001
From: Didier Raboud <odyx@debian.org>
Date: Tue, 27 Sep 2016 10:42:31 +0200
Subject: The upstream buildsystem touches files under debian/, forbid him to
 do so.

Last-Update: 2011-10-21
---
 Makefile.am | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index cf62595..5ccbb04 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -25,10 +25,6 @@ DIST_SUBDIRS = $(SUBDIRS)
 
 ACLOCAL_AMFLAGS = -I m4
 
-noinst_DATA = \
-	debian/control \
-	debian/rules
-
 debian/control: $(srcdir)/Makefile $(srcdir)/debian/control.in
 	rm -f $@ $@.tmp; \
 	sed -e 's|[@]PACKAGE_BUGREPORT[@]|$(PACKAGE_BUGREPORT)|' \
@@ -68,14 +64,6 @@ EXTRA_DIST = \
 	README \
 	README.ja \
 	bootstrap \
-	debian/changelog \
-	debian/compat \
-	debian/control.in \
-	debian/copyright \
-	debian/docs \
-	debian/postinst \
-	debian/postrm \
-	debian/rules.in \
 	epson-inkjet-printer-escpr.spec \
 	epson-inkjet-printer-escpr.spec.in \
 	lsb/lsb-rpm.spec \
